packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -82,51 +97,60 @@ export class GcdsCard {
     this.updateLang();
   }
 
-  private get hasCardFooter() {
-    return !!this.el.querySelector('[slot="footer"]');
+  private get renderDescription() {
```

Comment:
Great that the card can handle more flexible content, however this concept is not built into the [Card component in the Figma library](https://www.figma.com/design/mh2maMG2NBtk41k1O1UGHV/GC-Design-System?node-id=6322-16439&t=naPMzVAI6TCXr9Ak-4), curious how was this feature was handed off or communicated to be added to code?

Also what happens when the innerHTML and the description are both empty?

Reply:
This was a change mentioned within our requirements document made to provide better support based on [this issue](https://github.com/cds-snc/design-gc-conception/issues/927).

If there is no passed content or no description attribute, the card will render without a body of text and just have the card title(headline).

Reply:
Thank you, I was unaware of that issue. The main goal as I understand of the slot is to accommodate rich text, but slot opens up much more possibilities for what can be injected? If so is there a plan to limit it to rich text?

Reply:
Currently there is nothing being done to limit what can be passed

Reply:
Is the intent to leave it as a freestyle slot? If so adjustments may be needed on the Figma side

Reply:
I originally planned for it to be left freestyle similarly to how HTML lets you do whatever you want and you have to know what's the right thing to do.

But with that being said if we as a team want some validation into what is passed into the slot, I can start looking into that.

Reply:
Makes sense, just seems like team leadership wants to see a level of consistency in our components between dev and design. Also limiting our components to it's designed intent (behaviour, visual look and feel etc) is valuable as well. Anyways thanks Ethan, we can checkin with the team at the next available opportunity.

Reply:
As decided in design/dev sync, we will keep the slot open accepting the risks with that choice.